src/Trinity.C/src/Storage/MTHash/MTHash.CellInfo.cpp

```diff
@@ -75,18 +111,21 @@ namespace Storage
                 entryIndex = entry_index;
                 /////////////////////////
 
-                CellEntry entry = { updated_cell_offset, size };
+                CellEntry entry ={ updated_cell_offset, size };
                 std::atomic<int64_t> * CellEntryAtomicPtr = (std::atomic<int64_t>*) CellEntries;
```

Comment:
Should we put these into the if block below? It doesn't make sense to update the OUT arguments when something wrong happened.